fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -618,6 +619,12 @@ public static void loadJournal(GlobalStateMgr globalStateMgr, JournalEntity jour
                     LOG.debug("opcode: {}, tid: {}", opCode, state.getTransactionId());
                     break;
                 }
+                case OperationType.OP_UPSERT_TRANSACTION_STATE_BATCH: {
```

Comment:
Question: Is the addition of the new log here for correctness reasons, or for performance reasons?

Reply:
To make sure the updatement of visible version of the batch is atomic，so that client can not get a middle
visible version which tablet_meta is not exists.

Reply:
Got it, thx